docs/content/operations/api-reference.md

```diff
@@ -408,14 +450,25 @@ Retrieve a [task completion report](../ingestion/reports.html) for a task. Only
 
 Endpoint for submitting tasks and supervisor specs to the Overlord. Returns the taskId of the submitted task.
 
-* `druid/indexer/v1/task/{taskId}/shutdown`
+* `/druid/indexer/v1/task/{taskId}/shutdown`
 
 Shuts down a task.
 
-* `druid/indexer/v1/datasources/{dataSource}/shutdownAllTasks`
+* `/druid/indexer/v1/datasources/{dataSource}/shutdownAllTasks`
 
 Shuts down all tasks for a dataSource.
 
+* `/druid/indexer/v1/taskStatus`
+
+Retrieve list of task status objects for list of task id strings in request body.
+
+##### DELETE
+
+* `/druid/indexer/v1/pendingSegments/{dataSource}`
```

Comment:
Hmm, this was intended for internal use only. Do you think it would be useful for people too?

Reply:
Yeah, i wasn't thinking so hard about it. I think it's harmless to leave, but linked to `druid.coordinator.kill.pendingSegments.on` to mention that it can be done automatically via config.

Reply:
Thanks, 👍